### PR TITLE
[DX-915] Improve clarity on Rate Limiters

### PR DESCRIPTION
**Pull request by andyo-tyk**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link
https://deploy-preview-3776--tyk-docs.netlify.app/docs/nightly/basic-config-and-security/control-limit-traffic/rate-limiting/
<br>

### CODE REVIEW

File: tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md [hunk 1]
```diff
@@ -13,48 +13,61 @@ aliases:
 
 ## Rate Limiting Overview
 
-Also known as throttling, Tyk API will actively only allow a key to make `x` requests per `y` time period. This is very useful if you want to ensure your API does not get flooded with requests. See [Rate Limiting Concepts]({{< ref "getting-started/key-concepts/rate-limiting" >}}) for more information.
+You can configure Tyk Gateway to limit the rate of requests to an API which you might do, for example, if you want to ensure your API does not get flooded with requests. Rate limits in Tyk are configured using two parameters: allow `x` requests per `y` time period.
```
Comment (titpetric) [inline tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md]:
```suggestion
You can configure Tyk Gateway to limit the rate of requests to an API to ensure your API does not get flooded with requests. Rate limits in Tyk are configured using two parameters: allow `rate` requests per `per` time period.
```

File: tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md [hunk 2]
```diff
@@ -13,48 +13,61 @@ aliases:
...
+As explained in the [Rate Limiting Concepts]({{< ref "getting-started/key-concepts/rate-limiting" >}}) section, Tyk supports configuration of rate limits at both the API-Level and Key-Level for different use cases.
 
-## Setting up an API-Level Global Rate Limit
+The API-Level rate limit takes precedence over Key-Level, if both are configured for a given API, since this is intended to protect your upstream service from becoming overloaded. The Key-Level rate limits provide more granular control for managing access by your API clients.
 
-Adding an API-level global rate limit is what requires the least amount of effort and can be done as soon as you’ve created the API in Tyk. To apply a global rate limit you simply need to:
+## Configuring the rate limiter at the API-Level
 
-1. Navigate to the API you want to set the global rate limit on
+If you want to protect your service with an absolute limit on the rate of requests, regardless of where those requests come from, you can configure an API-level rate limit. You can do this from the API Designer in Tyk Dashboard as follows:
```
Comment (titpetric) [inline tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md]:
```suggestion
If you want to protect your service with an absolute limit on the rate of requests, you can configure an API-level rate limit. You can do this from the API Designer in Tyk Dashboard as follows:
```

File: tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md (hunk 1 above)
Comment (andyo-tyk) [reply tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md]:
```suggestion
You can configure Tyk Gateway to limit the rate of requests to an API to ensure your API does not get flooded with requests. Rate limits in Tyk are configured using two parameters: allow `rate` requests in any `per` time period (given in seconds).
```

File: tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md [hunk 3]
```diff
@@ -64,28 +77,28 @@ Check out the following video to see this being done.
...
-Yes, the rate limiter can be disabled for an API Definition by checking **Disable Rate Limits** in your API Designer, or by setting the value of `disable_rate_limit` to `true` in your API definition.
+Yes, the rate limiter can be disabled for an API Definition by selecting **Disable Rate Limits** in the API Designer, or by setting the value of `disable_rate_limit` to `true` in your API definition.
 
 Alternatively, you could also set the values of `Rate` and `Per (Seconds)` to be 0 in the API Designer.
 
 {{< note success >}}
 **Note**  
 
-Disabling the rate limiter at the global level does not disable the rate limiting at the key level.  Tyk will enforce the rate limit at the key level regardless of this setting.
+Disabling the rate limiter at the API-Level does not disable rate limiting at the Key-Level.  Tyk will enforce the Key-Level rate limit even if the API-Level limit is not set.
```
Comment (titpetric) [inline tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md]:
```suggestion
Disabling the rate limiter at the API-Level does not disable rate limiting at the Key-Level. Tyk will enforce the Key-Level rate limit even if the API-Level limit is not set.
```

Removed double space (pre-existing).

**Review by titpetric — APPROVED** [review]

File: tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md [hunk 4]
```diff
@@ -13,48 +13,61 @@ aliases:
 
 ## Rate Limiting Overview
 
-Also known as throttling, Tyk API will actively only allow a key to make `x` requests per `y` time period. This is very useful if you want to ensure your API does not get flooded with requests. See [Rate Limiting Concepts]({{< ref "getting-started/key-concepts/rate-limiting" >}}) for more information.
+You can configure Tyk Gateway to limit the rate of requests to an API to ensure your API does not get flooded with requests. Rate limits in Tyk are configured using two parameters: allow `rate` requests in any `per` time period (given in seconds).
```
Comment (andyo-tyk) [inline tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md]:
```suggestion
You can protect your upstream services from being flooded with requests by configuring rate limiting in Tyk Gateway. Rate limits in Tyk are configured using two parameters: allow `rate` requests in any `per` time period (given in seconds).
```

File: tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md [hunk 5]
```diff
@@ -13,48 +13,61 @@ aliases:
...
+
+1. Navigate to the Tyk policy for which you want to set the rate limit
 2. Ensure that API(s) that you want to apply rate limits to are selected
 3. Under **Global Limits and Quota**, make sure that **Disable rate limiting** is not selected and enter your **Rate** and **Per (seconds)** values
 4. **Save/Update** the policy
 
-## Setting up a Key-Level Per-API Rate Limit
+## Setting up a Key-Level Per-API rate limit
+
+If you want to restrict API clients to a certain rate of requests to a specific API you will also configure the rate limiter via the security policy, however this time you'll assign per-API limits. The allowance that you configure in the policy will be consumed by any requests made to that specific API using a key generated from that policy. Thus, if a policy grants access to an API with `rate=5 per=60` then three clients using keys generated from that policy will each independently be able to make 5 requests in any 60 second period before receiving the `HTTP 429 Too Many Requests` error.
```
Comment (dcs3spp) [inline tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md]:
```suggestion
If you want to restrict API clients to a certain rate of requests for a specific API you will also configure the rate limiter via the security policy. However this time you'll assign per-API limits. The allowance that you configure in the policy will be consumed by any requests made to that specific API using a key generated from that policy. Thus, if a policy grants access to an API with `rate=5 per=60` then three clients using keys generated from that policy will each independently be able to make 5 requests in any 60 second period before receiving the `HTTP 429 Too Many Requests` error. 
```

File: tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md [hunk 6]
```diff
@@ -64,28 +77,28 @@ Check out the following video to see this being done.
...
-## Set a rate limit on the session object (API)
+## Configuring the rate limiter on the session object
 
 All actions on the session object must be done via the Gateway API.
 
-1. Ensure that `allowance` and `rate` are set to the same value, this should be number of requests to be allowed in a time period, so if you wanted 100 requests every second, set this value to 100.
+1. Ensure that `allowance` and `rate` are set to the same value: this should be number of requests to be allowed in a time period, so if you wanted 100 requests every second, set this value to 100.
 
-2. Ensure that `per` is set to the time limit. Again, as in the above example, if you wanted 100 requests per second, set this value to 1. If you wanted 100 per 5 seconds, set this value to 5 etc.
+2. Ensure that `per` is set to the time limit. Again, as in the above example, if you wanted 100 requests per second, set this value to 1. If you wanted 100 per 5 seconds, set this value to 5.
```
Comment (dcs3spp) [inline tyk-docs/content/basic-config-and-security/control-limit-traffic/rate-limiting.md]:
```suggestion
2. Ensure that `per` is set to the time limit. Again, as in the above example, if you wanted 100 requests per second, set this value to 1. If you wanted 100 requests per 5 seconds, set this value to 5.
```

**Review by dcs3spp — APPROVED** [review]
PR lgtm Andy

**Pull request merged**